source/tutorial/install-auth-plugin-c.txt

```diff
@@ -72,6 +72,10 @@ Install ``mongosql_auth`` on Windows
 Install ``mongosql_auth`` on MacOS or Linux
 -------------------------------------------
 
+#. Download the `MySQL 5.7.x installer
+   <https://dev.mysql.com/downloads/mysql/>`_ and install the
+   MySQL shell.
```

Comment:
Same as [Line 52](https://github.com/mongodb/docs-bi-connector/pull/75/files#diff-4a4b9813426c98357ab712349294a397R52) in the other file.

Reply:
Updated.